src/extension/background-script/actions/allowances/add.ts

```diff
@@ -16,14 +21,23 @@ const add = async (message: MessageAllowanceAdd) => {
   if (allowance) {
     if (!allowance.id) return { error: "id is missing" };
 
-    await db.allowances.update(allowance.id, {
+    const eventData: AlbyEventBudgetUpdateDetails = {
+      type: AlbyEventBudgetType.UPDATE,
+      allowanceId: allowance.id,
+      event: AlbyEventType.BUDGET,
+    };
+
+    PubSub.publish("budget.success", eventData);
+
+    db.allowances.update(allowance.id, {
       enabled: true,
       imageURL: imageURL,
       name: name,
       remainingBudget: totalBudget,
       totalBudget: totalBudget,
     });
   } else {
+    // QUESTION: is it possible to reach this without having the site "enabled" before anyways (where an allowance is added already)?
```

Comment:
Asking this because `enable` is already adding an allowance via this, right:
https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/allowances/enable.ts/#L37-L62